### PR TITLE
Adding tests for required functions and passing them

### PR DESCRIPTION
**Pull request by SavaVuckovic**

### CODE REVIEW

File: caesar.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+function caesar(string, times) {
+  const result = [];
+
+  string.split('').forEach(character => {
+    if (character.match(/[a-z]/)) {
+      const charCode = character.charCodeAt() + times;
```
Comment (jmmerino) [inline caesar.js]: Maybe you can split the code in the "if's" in small functions?

File: calculator.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  multiply(x, y) {
+    return x * y;
+  }
+
+  divide(x, y) {
+    return x / y;
+  }
+}
+
+module.exports = new Calculator();
```
Comment (jmmerino) [inline calculator.js]: It's better if you export the class and then, when you are requiring it, you can create the instance one calculator with the `new`.

File: calculator.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const calculator = require('./calculator');
```
Comment (jmmerino) [inline calculator.test.js]: Here you can create the instance of the calculator after importing it.

File: calculator.test.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+test('performs subtract operation', () => {
+  expect(calculator.subtract(20, 10)).toBe(10);
+});
+
+test('performs multiply operation', () => {
+  expect(calculator.multiply(20, 10)).toBe(200);
+});
+
+test('performs divide operation', () => {
+  expect(calculator.divide(20, 10)).toBe(2);
```
Comment (jmmerino) [inline calculator.test.js]: Try to add tests with more values. Did you test your calculator if you divide by 0?

File: capitalize.test.js [hunk 5]
```diff
@@ -0,0 +1,9 @@
+const capitalize = require('./capitalize');
```
Comment (jmmerino) [inline capitalize.test.js]: Maybe you can add more tests: `Don’t forget to test punctuation!`

File: caesar.js (hunk 1 above)
Comment (SavaVuckovic) [reply caesar.js]: Done, code is much more understandable now ^^

**Pull request merged**